File: desktop-staff/reservation-winforms/DTO/reservation/ChangeTableRequest.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace reservation_winforms.DTO.reservation
{
    public class ChangeTableRequest
    {
        [JsonProperty("tableIds")]
        public List<long> TableIds { get; set; }

        [JsonProperty("reason")]
        public string Reason { get; set; }
    }
}

File: backend-api/reservation-api/src/main/java/com/hcmute/reservation/model/dto/reservation/ChangeTableRequest.java
package com.hcmute.reservation.model.dto.reservation;

import jakarta.validation.constraints.NotEmpty;
import lombok.Data;
import java.util.List;

@Data
public class ChangeTableRequest {
    @NotEmpty
    private List<Long> tableIds;

    private String reason; // optional, để log lại lý do đổi bàn
}